packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -243,6 +241,13 @@ export interface DiffOptions {
    * @default stderr
    */
   stream?: NodeJS.WritableStream;
+
+  /**
+   * Whether to fail with exit code 1 in case of diff
+   *
+   * @default false
+   */
+  fail?: boolean;
```

Comment:
This should be `failOnDifference`

Reply:
You me to open another PR to change this?

Reply:
It's ok. Let's just leave it like .